doc/ref_model/chapters/chapter03.md

```diff
@@ -3,11 +3,13 @@
 
 ## Table of Contents
 * [3.1 Model](#3.1)
-* [3.2 Virtual Resources](#3.2)
-  * [3.2.1 Tenant](#3.2.1)
-  * [3.2.2 Compute](#3.2.2)
-  * [3.2.3 Storage](#3.2.3)
-  * [3.2.4 Availability Zone](#3.2.4)
+* [3.2 Virtual Infrastructure Layer](#3.2)
+  * [3.2.1 Virtual Resources](#3.2.1)
+   * [3.2.1.1 Virtual Tenant](#3.2.1.1)
```

Comment:
I don't know what is a "Virtual Tenant". Maybe it is just copy ...
The section itself is just called "Tenant".

Reply:
@ulikleber you are right, there is. no such thing, copy and paste mistake.  I will change it

Reply:
@ulikleber, and in the text it was "Tenant"